libpod/boltdb_state.go

```diff
@@ -1234,7 +1234,7 @@ func (s *BoltState) networkModify(ctr *Container, network string, opts types.Per
 		netConnected := ctrNetworksBkt.Get([]byte(network))
 
 		if new && netConnected != nil {
-			return fmt.Errorf("container %s is already connected to network %q: %w", ctr.ID(), network, define.ErrNetworkConnected)
+			return fmt.Errorf("container %s is already connected to network %s: %w", ctr.ID(), network, define.ErrNetworkConnected)
```

Comment:
How is this more consistent? The error 2 lines below still uses `%q`.
I am in favor of consistency but I would argue this needs much broader vetting of error messages first and a consent from the team on what the right way way is, i.e. when to use %q vs %s, right now this seems rather arbitrarily chosen.

Reply:
I meant consistent between podman and podman-remote. Right now each one emits a different message, which is evil. I did not make that clear, and apologize for that.

I could instead add a conditional to the e2e test (if remote, Expect one thing, otherwise, Expect another). Or I could break this PR down into smaller parts. I'm tempted to do the latter. Suggestions welcome.

Reply:
How the hell do podman vs podman-remote generate different messages here? This error is coming straight from the db backend, are you sure this is remote vs local not boltdb vs sqlite?

I see what you are trying to do but IMO changing a few messages makes things worse unless we have a overall agreement first on what the right way to change it is. And IMO changing one message only while leaving others the same way makes podman error messages less consistent overall.

Reply:
Final (I hope) followup: I've removed my changes to the code base, and gone with `if/else IsRemote() or boltb` conditionals in the tests. I find that distasteful but I hope it will let us move forward.